web/client/utils/AnnotationsUtils.js

```diff
@@ -622,8 +621,11 @@ export const isAMissingSymbol = (style) => {
  * @return {boolean} true if it is a valid polygon, false otherwise
 */
 export const isCompletePolygon = (coords = [[[]]]) => {
-    const validCoords = coords[0].filter(AnnotationsUtils.validateCoordsArray);
-    return validCoords.length > 3 && head(validCoords)[0] === last(validCoords)[0] && head(validCoords)[1] === last(validCoords)[1];
+    if (coords && coords[0]) {
```

Comment:
please add or amend a unit test

Reply:
Updated unit tests for cases when polygon has 0 and 1 coordinate added.